tests/api/v1/bots/put.js

```diff
@@ -68,6 +68,23 @@ describe('tests/api/v1/bots/put.js >', () => {
     });
   });
 
+  it('Pass, put bot version & url', (done) => {
```

Comment:
Does it fail if the version or url is invalid?

Reply:
There isn't any version validation (we're expecting a string, and suggesting to follow semantic versioning 2.0.0).
For the url - it will fail if a malformed url is passed.

Reply:
take a look at /db/constants.versionRegex - Refocus uses that validation for the sample generator template version field